mito-ai/src/icons/Check.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const CheckIcon: React.FC<{
```

Comment:
My vote would be to have the buttons `Accept ⌘Y` and `Deny ⌘D` on Mac and `Accept CTRL+Y` and `Deny ⌘D`. My philosophy is information over design when it comes to things like this. 

But up to you for this version and we can iterate with user feedback! That's the joy of watching folks!